addons/armaos/CfgVehicles.hpp

```diff
@@ -104,7 +104,7 @@ class CfgVehicles
 		{
 			displayName = "Laptop";
 
-			closeState = 1;
```

Comment:
The animation is still played, because the open fnc is played regardless of state.
There maybe should be a default closedState which is set according to the objects default animation state